test/run_cargo_test.ps1

```diff
@@ -8,22 +8,29 @@ param (
     # Features to pass to cargo
     [string]$Features,
     # Options to pass to the cargo test command
-    [string]$TestOptions
+    [string]$TestOptions,
+    [string]$toolchain = "stable"
```

Comment:
In powershell this would generally be `$Toolchain`

Reply:
Honestly, this is a hacky solution that I was hoping to get changed to match the linux approach, but because I don't have the `pwsh` skills, I left it like this. If we're *happy* with linux doing
```
run_cargo_test.sh --nightly
```
while windows does
```
run_cargo_test.ps1 butterfly -toolchain "nightly" …
```
I can change it to
```
run_cargo_test.ps1 butterfly -Toolchain "nightly" …
```
But I think the better thing would be
```
run_cargo_test.ps1 butterfly -Nightly …
```
along with adding an equivalent for `get_nightly_toolchain` (before this PR: `get_current_toolchain`) for windows. Both versions should probably read that from a file rather than having it hard-coded in [`support/ci/shared.sh`](https://github.com/habitat-sh/habitat/blob/master/support/ci/shared.sh#L16). But because I didn't want to burn cycles doing `pwsh` that @mwrock could probably do much faster/better, I didn't.

If it's a simple thing, could you add a commit that makes that fix, @mwrock?

Reply:
done @baumanj